ui/app/src/state/reducers/pathNavigatorTree.ts

```diff
@@ -162,7 +162,7 @@ export function deleteItemFromState(tree: PathNavigatorTreeStateProps, targetPat
   tree.expanded = tree.expanded.filter(
     // If the parent is left without children, remove from expanded too.
     totalByPath[parentPath] === 0
-      ? (expandedPath) => expandedPath !== targetPath || expandedPath !== parentPath
+      ? (expandedPath) => expandedPath !== targetPath && expandedPath !== parentPath
```

Comment:
Condition was already there, but it needed to be an and instead of an or, since for any of the cases it needs to be filtered/removed from state